pyproject.toml

```diff
@@ -83,3 +83,12 @@ towncrier = "^23.6.0"
 [build-system]
 requires = ["poetry>=1;<1.1", "pip>=20"]
 build-backend = "poetry.masonry.api"
+
```

Comment:
nit: do we need two newlines here?

Reply:
Probably not, even though the formatter does not dislike it. Force-pushed change to just one newline